DependencyInjection/Configuration.php

```diff
@@ -822,6 +831,10 @@ public function getConfigTreeBuilder()
                             ->ifTrue(function ($v) { return 'native_mailer' === $v['type'] && (empty($v['from_email']) || empty($v['to_email']) || empty($v['subject'])); })
                             ->thenInvalid('The sender, recipient and subject have to be specified to use a NativeMailerHandler')
                         ->end()
+                        ->validate()
+                            ->ifTrue(function ($v) { return 'symfony_mailer' === $v['type'] && empty($v['email_prototype']) && (empty($v['from_email']) || empty($v['to_email']) || empty($v['subject'])); })
+                            ->thenInvalid('The sender, recipient and subject or an email prototype have to be specified to use a MailerHandler')
```

Comment:
```suggestion
                            ->thenInvalid('The sender, recipient and subject or an email prototype have to be specified to use the Symfony MailerHandler')
```

what about his?
as you validate first the `'symfony_mailer' === $v['type']`